src/EnergyPlus/SimulationManager.cc

```diff
@@ -741,14 +741,9 @@ namespace SimulationManager {
                                                                      state.dataIPShortCut->lAlphaFieldBlanks,
                                                                      state.dataIPShortCut->cAlphaFieldNames,
                                                                      state.dataIPShortCut->cNumericFieldNames);
-            {
-                auto const SELECT_CASE_var(Alphas(1));
-                if ((SELECT_CASE_var == "CONDUCTIONFINITEDIFFERENCE") || (SELECT_CASE_var == "CONDFD") ||
-                    (SELECT_CASE_var == "CONDUCTIONFINITEDIFFERENCEDETAILED") || (SELECT_CASE_var == "CONDUCTIONFINITEDIFFERENCESIMPLIFIED")) {
-                    CondFDAlgo = true;
-                } else {
-                }
-            }
+            static const std::array<std::string_view, 4> condFDTypes = {
+                "CONDUCTIONFINITEDIFFERENCE", "CONDFD", "CONDUCTIONFINITEDIFFERENCEDETAILED", "CONDUCTIONFINITEDIFFERENCESIMPLIFIED"};
+            CondFDAlgo = std::find(condFDTypes.begin(), condFDTypes.end(), Alphas(1)) != condFDTypes.end();
```

Comment:
I need to make the array a constexpr.

Reply:
I am holding off on making the getEnumerationValue change for now, but I'm not opposed to doing it.  I feel that this is a standard way of checking if a `std::array` simply contains an item, so it's not _necessary_ to use `getEnumerationValue`.  However, I understand we are trying to normalize some things as we go, so if it's worth it to change it to `getEnumerationValue`, I'm happy to do it.